code/go/0chain.net/blobbercore/challenge/protocol.go

```diff
@@ -301,6 +304,11 @@ func (cr *ChallengeEntity) LoadValidationTickets(ctx context.Context) error {
 }
 
 func (cr *ChallengeEntity) CommitChallenge(ctx context.Context, verifyOnly bool) error {
+	if time.Since(cr.CreatedAt) > config.Configuration.ChallengeCompletionTime+time.Second {
```

Comment:
why `+time.Second`?

Reply:
if this is for blobber server delay.. that's quite unpredictable
we don't know if it's ahead or behind..

Reply:
we need to ensure the times are same

Reply:
It was missed type.
I have deleted it in another PR